### ISSUE TITLE
Not working on Yun Shield

### ISSUE THREAD

**Issue by aimass** [arduino/yun-go-updater#4]
Is there a manual method? The OSX version is not working with Yun Shield

   AlexMac:osx alejandroimass$ sudo ./yun-go-updater 
   ...
   Using 10.0.1.7 as server address and 10.0.1.24 as board address, confirm? (Y, n)
   ...
   n 
   Enter server IP address: 10.0.1.7
   Enter board IP address: 10.0.1.5
   2018/05/04 22:05:43 No serial port suitable for updating Yun

**Comment by facchinm** [comment]
@aimass neither the updater or the sysupgrade image are ready for the yun shield. This will require a bit of extra work but we are open to contributions. In the meantime, please don't try to manually flash the image or you may end up with a bricked board.

**Comment by aimass** [comment]
@facchinm thanks a lot for your prompt reply. I may be able to help, can you summarize what would be required for Yun Shield support?
